index.html

```diff
@@ -255,11 +252,36 @@ <h3 class="one">Skills</h3>
               </div>
               <img src="images/Separator.png" class="Separator" width="6px" height="12px">
             </div>
-    
-            </div>
         </section>
 
+        <!--Setup the contact form section -->
+        
+        <footer class="footer-part">
+            <section>
+                <h3 class="contact-me">Contact me</h3>
+                <p class="call-me">If you have an application you are interested in developing, 
+                    a feature that you need built or a project that needs coding. 
+                    I’d love to help with it </p>
 
+                <form action="https://formspree.io/f/xqkwvblb" class="contact-form" method="post">
+                    <ul>
+                        <li>
+                        <input type="text" id="name" name="user_name" placeholder="Name" class="nem hint" required>
```

Comment:
- [ ] Please the  name field should accept up to 30 characters, please kindly fix that, as it is a requirement for this project and a form validation rule
![abdoform](https://user-images.githubusercontent.com/65852621/140047078-472e8da9-e5ce-497c-ad8b-c237c4a344a2.png)
.